docs/storage/develop/api-reference/index.mdx

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 1
+title: API reference information
+pagination_prev: null
+sidebar_label: API Reference
+description: Learn the API calls you need to know and how to use them with Momento Storage.
+---
+
+import { SdkExampleTabs } from "@site/src/components/SdkExampleTabs";
+// This import is necessary even though it looks like it's un-used; The inject-example-code-snippet
+// plugin will transform instances of SdkExampleTabs to SdkExampleTabsImpl
+import { SdkExampleTabsImpl } from "@site/src/components/SdkExampleTabsImpl";
+
+# API reference for Momento Storage
```

Comment:
doesn't need to happen right away, but should we start making api reference docs in the more stripe-ish style as used for [leaderboards](https://docs.momentohq.com/leaderboards/api-reference)?

Reply:
i completely forgot about this. I think I'd rather we do cache/topics/storage all at the same time for that, but i agree with you. maybe you can help me remember this in our next sprint planning.